datamodel/testdata.d/08_04_test_data_bkg.sql

```diff
@@ -45,6 +45,103 @@ INSERT INTO dcsa_im_v3_0.shipment_cutoff_time (
     DATE '2020-01-06'
 );
 
+INSERT INTO dcsa_im_v3_0.address (
+    id,
+    name,
+    street,
+    street_number,
+    floor,
+    postal_code,
+    city,
+    state_region,
+    country
+    )VALUES(
+    '8fecc6d0-2a78-401d-948a-b9753f6b53d5'::uuid,
+    'Lukas',
+    'Rohrdamm',
+    '81',
+    '5',
+    '32108',
+    'Bad Salzuflen Grastrup-hölsen',
+    'Nordrhein-Westfalen',
+    'Germany');
+
+INSERT INTO dcsa_im_v3_0.location (
+    id,
+    location_name,
+    latitude,
+    longitude,
+    un_location_code,
+    address_id,
+    facility_id
+    ) VALUES (
+    'c703277f-84ca-4816-9ccf-fad8e202d3b6',
+    'Hamburg',
+    '53.551° N',
+    '9.9937° E',
+    'DEHAM',
+    '8fecc6d0-2a78-401d-948a-b9753f6b53d5'::uuid,
+    (SELECT id FROM dcsa_im_v3_0.facility WHERE facility_name = 'DP WORLD JEBEL ALI - CT1' AND un_location_code = 'AEJEA' AND facility_smdg_code = 'DPWJA')
+    );
+
+-- Can be deleted once charge_type has been implemented properly
+INSERT INTO dcsa_im_v3_0.charge_type (
+    charge_type_code,
+    charge_type_name
+) VALUES (
+    'TBD',
+    'TO BE DELETED!'
+);
+
+INSERT INTO dcsa_im_v3_0.transport_document (
+    transport_document_reference,
+    place_of_issue,
+    issue_date,
+    shipped_onboard_date,
+    received_for_shipment_date,
+    number_of_originals,
+    issuer,
+    shipping_instruction_id,
+    declared_value_currency,
+    declared_value,
+    number_of_rider_pages
+) VALUES (
+    'be038e58-5365',
+    'c703277f-84ca-4816-9ccf-fad8e202d3b6',
+    DATE '2020-11-25',
+    DATE '2020-12-24',
+    DATE '2020-12-31',
+    12,
+    (SELECT id FROM dcsa_im_v3_0.carrier WHERE smdg_code = 'HLC'),
```

Comment:
Something seems wrong with this data (or it might be me) - This select part seems to refer to the shipping_instruction id (it is the 4th from the bottom and the forth attribute from the bottom above is shipping_instruction_id...?!?

Reply:
@HenrikHL  The select (fifth from bottom) is referring to issuer.